public/js/helpers/chartsHelpers.js

```diff
@@ -48,4 +51,28 @@ const fillMissingDates = (startDate, endDate, data) => {
     return data;
 };
 
-export { createPartialsList, formattedSeries, createYTotalsList, compareDates, fillMissingDates };
\ No newline at end of file
+// Components helpers
+
+
+const humanizeKeys = (obj, system) => ({ date: moment(obj.x).format('ddd, Do MMMM YYYY'), [`created_in_${system}`]: obj['y'] });
+
+const humanizeSeries = (series, system) => series.map(obj => humanizeKeys(obj, system));
+
+const unifiedSeries = (composerObject, inCopyObject) => _merge(composerObject ,inCopyObject);
+
+const replacer = (key, value) => value === null ? '' : value;
+
+const downloadCSV = (data, chartType) => {
+    //TODO make this chart specific when more chart types have been added
+    const absoluteData = data.absolute;
+    const merged = unifiedSeries(humanizeSeries(absoluteData[0].data, 'composer'), humanizeSeries(absoluteData[1].data, 'inCopy'));
+    const header = Object.keys(merged[0]);
+    let csv = merged.map(row => header.map(fieldName => JSON.stringify(row[fieldName], replacer)).join(','));
+    csv.unshift(header.join(','));
+    csv = csv.join('\r\n');
+    const blob = new Blob([csv], {type: 'text/csv;charset=utf-8'});
+
+    saveAs(blob, `${chartType}.csv`);
```

Comment:
could we add the time range to the filename to make it more unique?

Reply:
Yep, this seems sensible!